byron/chain/formal-spec/blockchain-spec.tex

```diff
@@ -548,18 +564,6 @@ \subsection{Counting signed blocks}
 $476^{\text{th}}$ block would be rejected. See Appendix
 \ref{apdx:calculating-t} for the background on this value.
 
-\begin{figure}[ht]
-  \emph{Protocol parameters}
-  \begin{equation*}
-    \begin{array}{r@{~\partialf~}l!{~\in~\ProtParams~}r}
-      \pp{stableAfter} & \mathbb{N} & \text{Chain stability parameter} \\
-      \pp{blockSignatureCountThreshold} & \left[\frac{1}{5}, \frac{1}{4}\right] & \text{Block signature count threshold} \\
-    \end{array}
-  \end{equation*}
-  \caption{Blockchain signature count protocol parameters}
-  \label{fig:defs:sigcnt}
-\end{figure}
```

Comment:
I'm not entirely seeing the reasoning behind this change. The aim was to split between updateable parameters (in the `pps` map) and non-updateable ones (which can be defined as abstract functions in much the same way as the other abstract functions). This change seems to do neither; instead, it just pushes the definition of these parameters up to the top level, where their context is much less understood.

I would rather either leave them as they are, or just add them to the "abstract functions" in this section, which would have the benefit of not changing much elsewhere.

Reply:
@ruhatch You may have opinions here?